concepts.md

```diff
@@ -7,118 +7,118 @@ title: SirixDB - Architecture and Concepts
 
 RecordPage
-: `RecordPage`s store the actual data. Currently, SirixDB stores 512 records in a record page. Each record page has a pointer to the previous record page, which stores the offset of the previous version of this record page. This pointer is crucial for our versioning algorithms, which have to retrieve several record pages (or record page fragments) to reconstruct a record page in-memory. Furthermore, records, which exceed a predefined size, are stored in so-called `OverflowPage`s and referenced in a record page.
+: `RecordPage`s store the actual data. Currently, SirixDB stores 512 records on a record page. Each record page has a pointer to the previous record page, which stores the offset of the previous version of this record page. This pointer is crucial for our versioning algorithms, which have to retrieve several record pages (or record page fragments) to reconstruct a record page in memory. Furthermore, records, which exceed a predefined size, are stored in so-called `OverflowPage`s and referenced on a record page.
 
 OverflowPage
 : `OverflowPage`s are used to store records, which exceeds a predefined size in bytes. When a transaction reads a record, SirixDB reconstructs the record page in-memory.
 
-However, potentially only a small fraction of records in the page have to be retrieved and reconstructed from byte-arrays in-memory for reads. SirixDB can delay this reconstruction until a transaction reads the overlong record.
+However, potentially only a small fraction of records on the page have to be retrieved and reconstructed from byte-arrays in-memory for reads. SirixDB can delay this reconstruction until a transaction reads the overlong record.
 
 ### Transaction commit
 
-The next figure depicts what happens during a transaction-commit.
+The next figure depicts what happens during a transaction commit.
 
 <div class="img_container">
 ![pageStructure](images/copy-on-write.png){: style="max-width: 100%; height: auto; margin: 0em"}
 </div>
 
-We assume that a read-write transaction modifies a record in the leftmost *RecordPage*. Depending on the versioning algorithm used by SirixDB, the modified record, as well as probably some other records of the page, are copied to a new page fragment. First, SirixDB stores all changes in an in-memory transaction (intent) log. Second, during a transaction commit, the page-structure of the current *RevisionRootPage* is serialized in a postorder traversal.
+We assume that a read-write transaction modifies a record in the leftmost *RecordPage*. Depending on the versioning algorithm used by SirixDB, the modified record, as well as probably some other records of the page, are copied to a new page fragment. First, SirixDB stores all changes in an in-memory transaction (intent) log. Second, during a transaction commit, the page structure of the current *RevisionRootPage* is serialized in a postorder traversal.
```

Comment:
> page structure

given that structure here describes the page ("the page, which is a structure"), it's probably better to use `page-structure` .